bin/merge_sces.R

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env Rscript
+
+# import libraries
+library(magrittr)
+library(optparse)
+library(SingleCellExperiment)
+
+# Set up optparse options
+option_list <- list(
+  make_option(
+    opt_str = c("--input_library_ids"),
+    type = "character",
+    help = "Comma separated list of library IDs corresponding to the libraries being integrated."
+  ),
+  make_option(
+    opt_str = c("--input_sce_files"),
+    type = "character",
+    help = "Comma separated list of input sce file paths corresponding to the sces being integrated."
+  ),
+  make_option(
+    opt_str = c("-o", "--output_sce_file"),
+    type = "character",
+    help = "Path to output RDS file, must end in .rds"
+  )
+)
+
+# Setup ------------------------------------------------------------------------
+
+# Parse options
+opt <- parse_args(OptionParser(option_list = option_list))
+
+# check that file extension for output file is correct 
+
+# list of paths to salmon files 
+input_sce_files <- unlist(stringr::str_split(opt$input_sce_files, ','))
+# pull library ids from list of 
+input_library_ids <- unlist(stringr::str_split(opt$input_library_ids, ','))
+
+# check that input files exist 
+
+# get list of sces
+sce_list <- purrr::map(input_sce_files, readr::read_rds)
+names(sce_list) <- input_library_ids
```

Comment:
If `library_ids` are brought in independently, we should still check that they match, or throw an error.

Reply:
I added this check earlier, and the way I did it allows me to delete a step here, as the names are already set (and purrr preserves them). I wonder if we should add a check that the read objects are SCEs though?
```suggestion
sce_list <- purrr::map(input_sce_files, readr::read_rds)
```